### DISCUSSION TITLE
Add A/B function switch

### DISCUSSION THREAD

**Discussion by mbs0ft** [dayanch96/BHTikTok-Plus#17] [Ideas]
Add A/B function switch, something like this [tweak from YouTube](https://github.com/PoomSmart/YTABConfig)

**Comment by dayanch96** [comment]
Hey, not sure if there are any a/b's in TikTok and most likely won’t be able to devote time to this since been busy lately

**Reply by mbs0ft** [reply]
На самом деле в TikTok много A/B фич, которые непонятно каким образом включаются, к примеру Picture in Picture или третья вкладка ленты только с фото. Так же фичи с разных регионов (к примеру отправка фото в комментариях доступна в некоторых регионах)

**Reply by dayanch96** [reply]
так они не в одном классе все собраны. все раскидано
